packages/bridge-ui/src/bridge/ETHBridge.ts

```diff
@@ -100,7 +100,7 @@ export class ETHBridge implements Bridge {
       opts.msgHash,
     );
 
-    if (messageStatus === MessageStatus.Done) {
+    if ([MessageStatus.Done, MessageStatus.Failed].includes(messageStatus)) {
```

Comment:
If not mistaken when `MessageStatus.Failed` is the status, then it is not processed yet. There need to be a 'releaseEther()'

Reply:
If Claimed called after releaseEther it's fine i guess - donno when / how automatized it is on the relayer side.

Reply:
We should then throw a different error message here?, In the UI when the status is Failed we show the Release button, claiming will no longer be available, so theoretically the user shouldn't be able to call this function when messageStatus is Failed... Let me make that change

Reply:
Ok, yes, so basically its kinda not possible for regular users, so then has not too much effect anyways.

Reply:
I made a change to throw different error when the message has failed. One could argue though that the message has failed and therefore already processed. @cyberhorsey, what do you think?

In V2 I'd like to use error codes instead of messages